micro_ros_setup/config/freertos/crazyflie21/create.sh

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
What's the rationale for copying this file, instead of modifying the existing scripts to be more general?